thrift/shared.thrift

```diff
@@ -1713,4 +1713,100 @@ struct RefreshWorkflowTasksRequest {
 
 struct FeatureFlags {
 	10: optional bool WorkflowExecutionAlreadyCompletedErrorEnabled
-}
\ No newline at end of file
+}
+
+enum CrossClusterTaskType {
+  StartChildExecution
+  CancelExecution
+  SignalExecution
+}
+
+enum CrossClusterTaskFailedCause {
+  DOMAIN_NOT_ACTIVE
+  WORKFLOW_ALREADY_RUNNING
+  WORKFLOW_NOT_EXISTS
+}
+
+struct CrossClusterTaskInfo {
+  10: optional string domainID
+  20: optional string workflowID
+  30: optional string runID
+  40: optional CrossClusterTaskType taskType
+  50: optional i16 taskState
```

Comment:
Another options it to define new taskTypes is `CrossClusterTaskType` and add the corresponding task response/request attributes